airflow/dag_processing/processor.py

```diff
@@ -149,8 +150,8 @@ def _run_file_processor(
 
         try:
             # redirect stdout/stderr to log
-            with redirect_stdout(StreamLogWriter(log, logging.INFO)), redirect_stderr(
-                StreamLogWriter(log, logging.WARN)
+            with redirect_stdout(StreamLogWriter(log, logging.INFO)), redirect_stderr(  # type: ignore
```

Comment:
Hmm. I thought this has been handled alredy by StremLogWriter implementing IO? what's the problem here?